internal/policies/client.go

```diff
@@ -171,10 +173,12 @@ func (f *Client) getAdmissionPolicies(ctx context.Context, namespace string) ([]
 
 // groupPoliciesByGVRAndLabelSelectorg groups policies by GVR.
 // If namespaced is true, it will skip cluster-wide resources, otherwise it will skip namespaced resources.
+// If the policy targets an unknown GVR or the policy server URL cannot be constructed, the policy will be counted as errored.
 func (f *Client) groupPoliciesByGVR(ctx context.Context, policies []policiesv1.Policy, namespaced bool) (*Policies, error) { //nolint:funlen
 	policiesByGVR := make(map[schema.GroupVersionResource][]*Policy)
 	auditablePolicies := map[string]struct{}{}
 	skippedPolicies := map[string]struct{}{}
+	erroredPolicies := map[string]struct{}{}
```

Comment:
`errored` can be a bit misleading, since we do keep track of policies that "error" (we send a POST request, but we get back something that isn't a 200 response).

In this case I would call them "misconfigured"

Reply:
I could create `erroredPolicies`, `misconfiguredPolicies`, but there's no certain way to know if policies are misconfigured, hence I prefer to be verbose. I see 3 possibilities:

1. A true error for policy-server URL, see https://github.com/kubewarden/audit-scanner/pull/312#discussion_r1665309373.
2. A misconfigured policy for the GVR issue, because `spec.rules` is incorrect. Technically misconfigured, I would like to count that as error for visibility. There's no easy way to ascertain this from 3.
3. A policy that was created for a CRD that isn't present anymore. Technically misconfigured. I find the chance of this happening is lesser than 2.

Reply:
All right, let's keep them into errored, otherwise we will end up with lots of possible outcomes -> confuse the end user.

@fabriziosestito do you agree with that?

Reply:
@flavio @viccuad I am ok with this

Reply:
resolving then!